doc/classes/TextEdit.xml

```diff
@@ -114,6 +114,12 @@
 				Starts a multipart edit. All edits will be treated as one action until [method end_complex_operation] is called.
 			</description>
 		</method>
+		<method name="begin_multicaret_edit">
+			<return type="void" />
+			<description>
+				Starts an edit for multiple carets. The edit must be ended with [method end_multicaret_edit]. Multicaret edits can be used to edit text at multiple carets and delay merging the carets until the end, so the caret indexes aren't affected immediately. [method begin_multicaret_edit] and [method end_multicaret_edit] can be nested, and the merge will happen at the last [method end_multicaret_edit].
```

Comment:
Is there any point in nesting? This seems like just a toggle between "no caret merging mode" and "caret merging mode" to me.

Reply:
Yes, since some methods like `insert_text_at_caret()` and `delete_selection()` need to be able to operate on all carets and on just one as part of another edit, so it's useful to be able to nest them.
While I don't expect users to need to call functions that operate on multiple carets within a loop, they still might. Also there's no reason to not have it, since we still need to know when a multicaret edit starts and ends.

I could remove the nesting part of the docs, to not encourage it?